code/client/munkilib/updatecheck/core.py

```diff
@@ -174,6 +174,16 @@ def check(client_id='', localmanifestpath=None):
                     localonlymanifest, 'managed_uninstalls') or []
                 for item in localonlyuninstalls:
                     analyze.process_removal(item, cataloglist, installinfo)
```

Comment:
Can you fix this line while you're in here? This line doesn't match the style above it or the code you're adding.
```python
                    dummy_result = analyze.process_removal(
                        item,
                        cataloglist,
                        installinfo
                    )
```